pkg/contexts/ocm/transfer/frontend.go

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Open Component Model contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package transfer
+
+import (
+	"github.com/open-component-model/ocm/pkg/common"
+	"github.com/open-component-model/ocm/pkg/contexts/ocm"
+	"github.com/open-component-model/ocm/pkg/contexts/ocm/transfer/transferhandler"
+)
+
+// TransferComponentVersionWithHandler uses the specified transfer handler to control
+// the transfer process.
+func TransferComponentVersionWithHandler(pr common.Printer, cv ocm.ComponentVersionAccess, tgt ocm.Repository, handler transferhandler.TransferHandler) error {
+	return TransferVersion(pr, nil, cv, tgt, handler)
+}
+
+// TransferComponentVersion uses the transfer handler based on the given options to control
+// the transfer process. The default handler is the standard handler.
+func TransferComponentVersion(printer common.Printer, cv ocm.ComponentVersionAccess, tgt ocm.Repository, optlist ...transferhandler.TransferOption) error {
```

Comment:
The repository is the intended target repository for the transfer, how could this be inferred? FRom which other given information?

The printer can be set to nil. It is intended by catch processing information, typically sent to a user.
r could this add this as additional option for the option list. I'll try to do this.

The problem hereby is, that the options so far are only used for transfer handler options. But this would a free floating options, which has to be considered by the convenience methods, only

Reply:
I've moved this argument to an additional possibility for an option, The default now is /dev/null.